welcome-message/main.scss

```diff
@@ -1,15 +1,16 @@
 @import 'o-visual-effects/main';
+@import 'o-icons/main';
 
 @if nUiApply('welcomeMessage') {
 
 	@include nUiMaybeCriticalStart('welcomeMessage');
 
-	.n-banner-message,
+	.n-welcome-banner,
 	.n-welcome-message {
 		position: relative;
 		padding: 16px 12px;
 		// not in o-colors ಠ_ಠ
-		background: #FADBCB;
+		background: #fadbcb;
```

Comment:
Sometimes we have to do bad things... I don't remember why this is so I'm afraid.

Moving into o-colors would make sense if we wish to use it more widely (though none of the `pink-` colours are actually, currently pink =S)